presto-main/src/main/java/io/prestosql/sql/planner/iterative/rule/TransformExistsApplyToCorrelatedJoin.java

```diff
@@ -151,19 +151,19 @@ public Result apply(ApplyNode parent, Captures captures, Context context)
                 assignments.build()));
     }
 
-    private PlanNode rewriteToDefaultAggregation(ApplyNode parent, Context context)
+    private PlanNode rewriteToDefaultAggregation(ApplyNode applyNode, Context context)
```

Comment:
why this name is better?

Reply:
For consistency reason. That name is used in the other method.